tests/Cluster.py

```diff
@@ -67,10 +74,13 @@ def __init__(self, walletd=False, localCluster=True, host="localhost", port=8888
         """
         self.accounts={}
         self.clusterID=clusterID
-        if clusterID != 0:
-            port += clusterID * 2000
-            Cluster.__BiosPort = 8788 + clusterID * 2000
-            Cluster.__BootScript = "bios_boot_cluster" + str(clusterID) + ".sh"
+        Utils.clusterID=clusterID
```

Comment:
Why is this stopping over `Utils.clusterID`?

If `Utils.getNodeDataDir` and `Utils.getNodeConfigDir` need the cluster ID, then it should be passed in as an argument to the function.

Reply:
I'll only put clusterID into Utils only, which makes thing much easier and can be used in Cluster, WalletMgr, and Node.